src/services/textService.ts

```diff
@@ -0,0 +1,19 @@
+import {
+    POLISH_COMMON_WORDS,
+    LETTERS_AND_PUNCTUATION_REGEX,
+} from '../utils/constants';
+
+export const textToArray = (text: string) => {
+    return text.match(LETTERS_AND_PUNCTUATION_REGEX) as string[];
+};
+
+export const blurWords = (words: string[]) => {
```

Comment:
sorry, missed that earlier - now blurWords lost its context as we are doing something else here. Something like createWordObjects would be better, however it is also not perfect. Bonus points if you can think of better name ;)

Reply:
createWordsToGuessObjects I think would be ok, becouse it explains the type that its returning (WordToGuess)